src/fmu/tools/rms/generate_bw_per_facies.py

```diff
@@ -0,0 +1,102 @@
+import copy
+from typing import Dict, List
+
+import numpy as np
+import xtgeo
+
+
+def create_bw_per_facies(
+    project,
+    grid_name: str,
+    bw_name: str,
+    original_petro_log_names: List[str],
+    facies_log_name: str,
+    facies_code_names: Dict[int, str],
+    debug_print: bool = False,
+) -> None:
+    """Script to be used as RMS python job to create new petrophysical
```

Comment:
Perhaps, instead of "Script to be used...":
"Function to be imported and applied in a RMS python job..."